### PR TITLE
Admins management

### PR DESCRIPTION
**Pull request by johangv**
Elimination and creation of admins

https://drive.google.com/file/d/11dUv9e0B2fT8-NReOhuIy7PVjrR3H7UT/view?usp=sharing

### CODE REVIEW

File: lib/features/home/home_screen.dart [hunk 1]
```diff
@@ -55,12 +55,27 @@ class HomeScreen extends StatelessWidget {
...
+                        }
+                      },
+                      orElse: () => Container())
                 ],
               ),
             ),
             body: const Column(
               children: [
-                Text('Fiura'),
+                // Text('Fiura'),
```
Comment (oscarshaitan) [inline lib/features/home/home_screen.dart]: delete this

File: lib/features/admins/presentation/pages/admins_screen.dart [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                child: const Icon(Icons.add),
+              ),
+              body: adminList.isNotEmpty
+                  ? FutureBuilder<void>(
+                      future: Future.wait(
+                        adminList.map(
+                          (admin) =>
+                              precacheImage(NetworkImage(admin.photo), context),
+                        ),
+                      ),
```
Comment (oscarshaitan) [inline lib/features/admins/presentation/pages/admins_screen.dart]: I have been thinking about this implementation and my issue is that iof the page has 1000 photos the view has to wait until precache all the 1000 to show the view. so could be slow in certain cases and is also possible that you are loading 90% stuff that the user will no see

**Review by oscarshaitan — CHANGES_REQUESTED** [review]

File: lib/features/home/home_screen.dart (hunk 1 above)
Comment (johangv) [reply lib/features/home/home_screen.dart]: Done

File: lib/features/admins/presentation/pages/admins_screen.dart (hunk 2 above)
Comment (johangv) [reply lib/features/admins/presentation/pages/admins_screen.dart]: fixed

**Pull request merged**